x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -153,7 +151,7 @@ export class HeadlessChromiumDriverFactory {
 
       // HeadlessChromiumDriver: object to "drive" a browser page
       const driver = new HeadlessChromiumDriver(page, {
-        inspect: this.browserConfig.inspect,
+        inspect: !!this.browserConfig.inspect,
```

Comment:
Good catch, do folks commonly set truthy values here? `inspect: 1` for instance?

Reply:
This had to be adjusted because `browserConfig.inspect` could be undefined as it can't be set for non-dev mode. Typescript needed the value to be coerced to boolean.